### PR TITLE
Fix some Edge values

### PR DESCRIPTION
**Pull request by FremyCompany**
I also planned to change `-webkit-line-clamp` too but apparently someone already reported that change before.

### CODE REVIEW

File: features-json/webp.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
       "15":"n",
       "16":"n",
       "17":"n",
-      "18":"n"
+      "18":"d"
```
Comment (HM100) [inline features-json/webp.json]: Change **d** to **n d** so as to be correct

Comment (HM100) [reply features-json/webp.json]: @FremyCompany Any way of fixing that ones I gave? It is needed to be done by now.

**Comment by FremyCompany** [comment]
@HM100 Change done!

**Comment by HM100** [comment]
Thanks!

**Comment by Fyrd** [comment]
Thanks! One change required is that the support for `text-size-adjust` should be prefixed, but I'll take care of that.

**Pull request merged**